stellargraph/core/graph.py

```diff
@@ -871,6 +871,10 @@ def to_networkx(
         """
         Create a NetworkX MultiGraph or MultiDiGraph instance representing this graph.
 
+        Args:
+            feature_name (str, optional): the name of the attribute to use to store a node's feature
```

Comment:
I wonder if this docstring is a bit misleading if the user decides to pass in a value without using keyword arguments, since it looks like `feature_name` is the first argument to this method, when it's actually the 4th.

Reply:
I guess so 🤷‍♂ 

I added docs for the other args.